tests/core/pyspec/eth2spec/test/helpers/genesis.py

```diff
@@ -15,16 +15,31 @@
 def build_mock_validator(spec, i: int, balance: int):
     active_pubkey = pubkeys[i]
     withdrawal_pubkey = pubkeys[-1 - i]
-    # insecurely use pubkey as withdrawal key as well
-    withdrawal_credentials = spec.BLS_WITHDRAWAL_PREFIX + spec.hash(withdrawal_pubkey)[1:]
+    if is_post_electra(spec):
+        if balance > spec.MIN_ACTIVATION_BALANCE:
+            # use compounding withdrawal credentials if the balance is higher than MIN_ACTIVATION_BALANCE
+            withdrawal_credentials = (
+                spec.COMPOUNDING_WITHDRAWAL_PREFIX
+                + b'\x00' * 11
+                + spec.hash(withdrawal_pubkey)[12:]
+            )
+        else:
+            # insecurely use pubkey as withdrawal key as well
+            withdrawal_credentials = spec.BLS_WITHDRAWAL_PREFIX + spec.hash(withdrawal_pubkey)[1:]
+        max_effective_balace = spec.MAX_EFFECTIVE_BALANCE_ELECTRA
```

Comment:
```suggestion
        max_effective_balance = spec.MAX_EFFECTIVE_BALANCE_ELECTRA
```

Reply:
ooops. will fix it after the release. 

Thanks for the review!